.github/workflows/sync-version.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  pull-request:
```

Comment:
I liked your previous suggestion `sync-branches`, it was less generic. I think we will easily forget what `pull-request` means.